niceml/data/datasets/dfdataset.py

```diff
@@ -91,7 +199,19 @@ def get_data(self, start_idx: int, end_idx: int):
         return self.get_data_from_idx_list(cur_indexes)
 
     def get_data_by_key(self, data_key):
-        return self.dataframe.loc[self.dataframe[self.id_key] == data_key]
+        """
+        The get_data_by_key function takes a data_key as an argument and returns the row of
```

Comment:
Too detailed information for a docstring, in my opinion.
Suggestion:
Returns all rows of the data, whose 'id_key' matches the 'data_key'.

Reply:
Done